bin/cmd.js

```diff
@@ -202,34 +202,28 @@ if (['info', 'create', 'download', 'add', 'seed'].includes(command) && argv._.le
 } else if (command === 'download' || command === 'add') {
   const torrentIds = argv._.slice(1)
 
-  if (torrentIds.length > 1) {
-    handleMultipleInputs(torrentIds)
-  }
+  if (torrentIds.length > 1) handleMultipleInputs(torrentIds)
+  numTorrents = torrentIds.length
```

Comment:
Is there any way to avoid duplicating this code? Seems like it's needed for most of these cases